stories/atoms/components/languageSwitcher/LanguageSwitcher.stories.mdx

```diff
@@ -0,0 +1,63 @@
+import { ArgsTable, Meta, Story, Canvas, Preview, Props } from '@storybook/addon-docs'
+import { LanguageSwitcher } from 'ui/atoms/languageSwitcher/LanguageSwitcher'
+import { ThemeProvider } from 'ui/atoms/theme/ThemeProvider'
+import { ThemeSwitcher } from 'ui/atoms/theme/ThemeSwitcher'
+
+<Meta
+  title="Atoms/LanguageSwitcher"
+  component={LanguageSwitcher}
+  decorators={[
+    Story => (
+      <ThemeProvider>
+        <div className="component-story-main language-switcher">
+          <ThemeSwitcher />
+          <Story />
+        </div>
+      </ThemeProvider>
+    )
+  ]}
+/>
+
+# Language Switcher
+
+> Allows the user to change the application language.
+
+[![stability-unstable](https://img.shields.io/badge/stability-unstable-yellow.svg)](https://github.com/emersion/stability-badges#unstable)
+
+## Overview
+
+### Description
+
+`LanguageSwitcher` component is used to offer mutli-languages support to the user in order to improve his experience on the application.
```

Comment:
Found a misspelled word 😄   The rest is just suggestions
```suggestion
`LanguageSwitcher` is used to offer multi-language support to users in order to improve their experience on the application.
```

Reply:
Good catch! as well as for the suggestion!